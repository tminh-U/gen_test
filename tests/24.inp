15
5 16 9 18 9 16 2 2 3 18 4 8 7 2 17 
4
9 11
6 15
5 8
3 15
