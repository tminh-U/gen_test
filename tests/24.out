6
115
7
137
